test/e2e/framework/node/resource.go

```diff
@@ -377,6 +377,41 @@ func GetRandomReadySchedulableNode(c clientset.Interface) (*v1.Node, error) {
 	return &nodes.Items[rand.Intn(len(nodes.Items))], nil
 }
 
+// GetSubnetPrefix gets first 2 number of an IP in the node subnet. [IPv4]
```

Comment:
nit, SubnetPrefix assumin is a /16 :upside_down_face:

Reply:
right, I don't see how we can get subnet from an IP or even all nodes' IPs. is cloud provider vendor-ed into e2e? 
/hold until this is fully understood.

Reply:
Or does the test always run with a /16 subnet? if so can we please update the test doc?

Reply:
I updated the test doc. Yes, this test assumes a /16 subnet just like it used to do.

Reply:
@khenidak this area has a lot of tech debt, just bear in mind this test was much worse before this PR
https://github.com/kubernetes/kubernetes/pull/109413/files#r847224114

I suggest to merge now and iterate on the subnet assumption later ... I defer to you to unhold

Reply:
@khenidak could you take a look at @aojea 's suggestion? Now v1.24 is released.